src/entity/Keyword.ts

```diff
@@ -23,6 +23,7 @@ export class Keyword {
   value: string;
 
   @Column({
+    type: 'text',
```

Comment:
swc couldn't guess the type by itself hence it failed to boot

Reply:
Doesn't hurt to explicitly set the type